### ISSUE TITLE
ADT assay slot lost after subset Seurat object

### ISSUE THREAD

**Issue by gt7901b** [satijalab/seurat#3394]
Hi:

Before subset, my seurat object has 
@ assays      :List of 3, RNA, HTO, ADT
after subset, seurat_obj.small <- subset(seurat.obj, downsample = 300)
only  RNA and HTO left. 
ADT disappeared. 
Any possible reasons?

thanks

**Comment by gt7901b** [comment]
I found that my subset function is from BioCGenerics, not from seurat. But when I force 
Seurat::subset(x = seurat.obj, features = VariableFeatures(object = other))
I got this error 
Error: 'subset' is not an exported object from 'namespace:Seurat'
Traceback:

1. Seurat::subset
2. getExportedValue(pkg, name)
3. stop(gettextf("'%s' is not an exported object from 'namespace:%s'", 
 .     name, getNamespaceName(ns)), call. = FALSE, domain = NA)

R4.0, Seurat 3.2
